docs/iris/src/developers_guide/gitwash_dumper.py

```diff
@@ -57,16 +57,19 @@ def filename_search_replace(sr_pairs, filename, backup=False):
     ''' Search and replace for expressions in files
 
     '''
-    in_txt = open(filename, 'rt').read(-1)
+    with open(filename, 'rt') as in_fh:
```

Comment:
I don't mind what happens here - we can keep the changes or not. As an aside, it'd be nice to push them upstream either way.